xwalk.gyp

```diff
@@ -632,8 +634,11 @@
       ],
       'copies': [
```

Comment:
In above `generate_xwalk_string` action, .pak files are generated into subfolder of "gen" first. Then in target "copies" we decide what is the copy destination.
I think it is ok to folllow the old logic,  no matter what the new destination is.